dicss.js

```diff
@@ -72,7 +72,12 @@
 	window.DICSS = {
 		"putIn": function(selector, properties) {
 			if (properties !== void 0) {
-				addRule.apply(this, arguments);
+				var consent = confirm("Are you sure you consent to the insertion of DICSS?");
+				if (consent == true) {
```

Comment:
Shouldn't it be `===`? [Beware of `==`](https://dorey.github.io/JavaScript-Equality-Table/).
Not required here but better to hold good coding standarts.